### PR TITLE
HDDS-5563. feature/Observability.md translation

### PR DESCRIPTION
**Pull request by iamabug**
## What changes were proposed in this pull request?

doc translation

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-5563

## How was this patch tested?

hugo server

image

### CODE REVIEW

File: hadoop-hdds/docs/content/feature/Observability.zh.md [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+## ozone insight
+
+Ozone insight 是一把查看 Ozone 集群当前状态的瑞士军刀。它可以展示特定组件的日志、指标和配置。
+
+使用 `ozone insight list` 命令查看可选组件：
+
+```shell
+> ozone insight list
+
+Available insight points:
```
Comment (iamabug) [inline hadoop-hdds/docs/content/feature/Observability.zh.md:91]: I chose not to translation these (shell command and output), but I could use a second opioion.

File: hadoop-hdds/docs/content/feature/Observability.zh.md [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+
+根据[此处文档](https://github.com/jaegertracing/jaeger-client-java/blob/master/jaeger-core/README.md)，Jaeger 客户端可以通过环境变量进行配置，例如：
+
+```shell
+JAEGER_SAMPLER_PARAM=0.01
+JAEGER_SAMPLER_TYPE=probabilistic
+JAEGER_AGENT_HOST=jaeger
+```
+
+此配置会记录 1% 的请求来限制性能开销。更多有关 jaeger 采样的信息[请查阅文档](https://www.jaegertracing.io/docs/1.18/sampling/#client-sampling-configuration)
```
Comment (iamabug) [inline hadoop-hdds/docs/content/feature/Observability.zh.md:80]: The English sentence is `This configuration will record 1% of the requests to limit the performance overhead`. It is easy to translate but not easy to understand for me. @elek Could you explain a little bit about `limit the performance overhead`?

File: hadoop-hdds/docs/content/feature/Observability.zh.md [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+
+## ozone insight
```
[the remark this answers is not in the document]
Comment (iamabug) [reply hadoop-hdds/docs/content/feature/Observability.zh.md:82]: This problem exists in many places, English docs use both Ozone and ozone. If we believe this needs to be fixed, maybe we can open a separate ticket to do this.

[the remark this answers is not in the document]
Comment (sky76093016) [reply hadoop-hdds/docs/content/feature/Observability.zh.md:82]: This is indeed a problem that needs to be improved.

**Comment by adoroszlai** [comment]
@iamabug @sky76093016 thanks for the patch and review.  Seems like there is some merge conflict.

**Pull request closed without merging**